src/components/TitlesCarousel/TitlesCarousel.js

```diff
@@ -43,6 +43,9 @@ export const MovieCarousel = async ({ movies, id, btnName, route }) => {
                     next.style.display = 'flex';
                 }
             });
+            if (movies.length <= 5) {
```

Comment:
Wrong, on mobile there is only 3 tiles visible, that should be dynamic

Reply:
done